peer/LocalData.py

```diff
@@ -15,6 +15,9 @@ class LocalData:
 	# ('ipv4', 'ipv6', 'port')
 	superpeer_candidates = list()
 
+	# ('file_name', 'file_md5')
+	files = list()
```

Comment:
@xBlue0 @jhonrambo93 @GiovanniFiorini io voto per ```shared_files```. Siccome abbiamo la ```peer_files```, ```shared_files``` mi sarebbe più chiaro in lettura del codice